test/visual-regression/datepicker.spec.ts

```diff
@@ -30,24 +30,36 @@ test("datepicker default change calendar language", async ({ page }) => {
   await page.goto(
     "/iframe.html?args=&id=components-form-date-datepicker--datepicker&viewMode=story",
   );
+
+  await expect(page.getByPlaceholder("dd.mm.yyyy")).toBeVisible();
   await page.getByTestId("fudis-datepicker-1").fill(date);
   await page.getByTestId("fudis-button-2").click();
+  await expect(page.getByPlaceholder("pp.kk.vvvv")).toBeVisible();
   await expect(page.getByText("Current language: fi")).toBeVisible();
   await page.getByTestId("fudis-datepicker-1-calendar-icon-toggle").click();
   await page.waitForSelector(".mdc-button__label");
   await expect(page).toHaveScreenshot("to-finnish.png", { fullPage: true });
   await page.keyboard.press("Escape");
   await expect(page.getByText("Choose your favourite date.")).toBeVisible();
+
+  await page.keyboard.press("Meta+A"); // Clear selection so that we can assert correct placeholder
+  await page.keyboard.press("Backspace");
```

Comment:
I don't see any point for having this?  Tests will pass even when these are removed?